app/views/diaper_drive_participants/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<% content_for :title, "#{current_organization.name} - Agencies - Diaper Drive Participants - #{@diaper_drive_participant.name}" %>
+
+<h1><%= @diaper_drive_participant.name %></h1>
+
+<ul>
+<% if @diaper_drive_participant.phone.present? %>
+  <li><%= @diaper_drive_participant.phone %></li>
```

Comment:
Not sure if you need the conditional views here if you are validating the presence of diaper drive participant phone and email?